### DISCUSSION TITLE
Custom sort function for operations in .NET

### DISCUSSION THREAD

**Discussion by woubuc** [scalar/scalar#3948] [Ideas]
Currently the sort options for operations (at least in C#) are either alphabetic by operation name, or alphabetic by method name. I would like to sort the operations in a custom order (particularly I want the methods GET, POST, PATCH, DELETE in that order) but this doesn't seem possible at the moment.

I would be happy with just the option to inject a javascript sorter function as a string, if full C# integration isn't feasible.

**Comment by xC0dex** [comment]
Hey @woubuc, the .NET package is based on the [HTML integration](https://github.com/scalar/scalar/blob/main/documentation/integrations/html.md) which uses a JSON string to configure the options. Unfortunately, a custom sort function is not available here.
